src/core/thread/mesh_forwarder.cpp

```diff
@@ -1874,11 +1874,11 @@ void MeshForwarder::LogMessage(MessageAction       aAction,
 #if OPENTHREAD_CONFIG_DELAY_AWARE_QUEUE_MANAGEMENT_ENABLE
     case kMessageMarkEcn:
 #endif
+    case kMessageReassemblyDrop:
```

Comment:
Maybe this is still going to cause log spam. When the Reassembly times out repeatedly.